libs/s25main/ingameWindows/iwMinimap.cpp

```diff
@@ -41,7 +44,16 @@ iwMinimap::iwMinimap(IngameMinimap& minimap, GameWorldView& gwv)
     // Fenster vergrößern/verkleinern
     AddImageButton(4, DrawPoint(0, 0), BUTTON_SIZE, TextureColor::Grey, LOADER.GetImageN("io", 109));
 
-    Resize(GetSize());
+    Resize(extended ? MINIMAP_SIZE_BIG : MINIMAP_SIZE);
+}
+
+iwMinimap::~iwMinimap()
```

Comment:
Missing the parent call which saves the position

Reply:
I disagree. Parent destructors are always called automatically in c++.